docs/pages/access-controls/access-requests/resource-requests.mdx

```diff
@@ -39,17 +39,17 @@ It is not recommended to enable Resource Access Requests by setting any
 upgraded to version 10.
 </Admonition>
 
-## Step 1/8. Create the requester role
+## Step 1/8. Create the my-requester role
```

Comment:
Should we instead just ask the user to assign the new preset `requestor` role rather than create another one?

Reply:
I was wondering about that. I think there's some use in having this walk through of creating review/request roles from scratch, so I decided to leave this.

Reply:
I dunno, I think there are plenty of other places we can teach a user how to create a role from scratch. The goal of this guide is to see access requests in action. If your new changes allow us to get the user to that outcome faster, I think we should do it.